src/main/java/com/unb/tracker/model/User.java

```diff
@@ -0,0 +1,71 @@
+package com.unb.tracker.model;
+
+import javax.persistence.*;
+
+@Entity // This tells Hibernate to make a table out of this class
+public class User {
+    @Id
+    @GeneratedValue(strategy=GenerationType.AUTO)
+    private Integer id;
+
+    private String username;
+
+    private String email;
+
+    private String password;
+
+    @Transient
+    private String passwordConfirm;
+
+    private boolean hasExtendedPrivileges;
+
+    public Integer getId() {
+        return id;
+    }
+
+    public void setId(Integer id) {
+        this.id = id;
+    }
+
+    public String getUsername() {
+        return username;
+    }
+
+    public void setUsername(String username) {
+        this.username = username;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String email) {
+        this.email = email;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
+    @Transient
```

Comment:
Can I get a quick explaination of this annotation?

Reply:
Transient basically tells Hibernate not to store this in the database. The methods and fields are basically ignored :) I added this because the passwordConfirm field was being stored in the database in plain text, which was obviously a bad idea.